vega_sim/api/data.py

```diff
@@ -304,12 +305,26 @@ class LiquidityProviderFeeShare:
     average_score: float
 
 
+@dataclass(frozen=True)
+class LiquiditySLA:
```

Comment:
Small nitpick but could we have this dataclass match the corresponding proto `LiquidityProviderSLA`?

Reply:
Renamed now